spotlight/losses.py

```diff
@@ -112,8 +118,8 @@ def hinge_loss(positive_predictions, negative_predictions, mask=None):
         The mean value of the loss function.
     """
 
-    loss = torch.clamp(negative_predictions -
-                       positive_predictions +
+    loss = torch.clamp((negative_predictions -
+                        positive_predictions) * weights +
```

Comment:
Again, I think the `weights` term should be outside the outermost parenthesis.

So it should be equivalent to
```python
loss = torch.clamp(negative_predictions - positive_predictions + 1.0), 0.0)
loss *= weights
```